### PR TITLE
feat: add godwoken v0 support

### PR DESCRIPTION
**Pull request by zeroqn**
- new `START_GODWOKEN_V0` environment variable, set true to start godwoken v0.10
- fix kicker `get-balance`
- new kicker `deposit-v0`
- new kicker `withdraw-v0-to-v1`

### CODE REVIEW

File: kicker [hunk 1]
```diff
@@ -235,31 +246,97 @@ function deposit() {
...
             --config-path /config/godwoken-config.toml \
             --scripts-deployment-path /config/scripts-deployment.json \
             --eth-address $ethaddr" \
-            godwoken)
+            godwoken 2>&1)
     echo "$output" | grep -oE '0x.*'
 }
 
+# TODO: Test withdraw v0 to v1, remove after abandon v0
+# @example deposit 1000
```
Comment (keroro520) [inline kicker:276]: deposit-v0

File: kicker [hunk 2]
```diff
@@ -235,31 +246,97 @@ function deposit() {
...
+function deposit-v0() {
+    amount=${1:?"\"$EXECUTABLE deposit\" requires amount as 1nd argument"}
```
Comment (keroro520) [inline kicker:278]: deposit-v0

File: kicker [hunk 3]
```diff
@@ -235,31 +246,97 @@ function deposit() {
...
+                    --capacity $amount" \
+            godwoken-v0 && echo "Deposit finished" && break
+        sleep 6 && echo "Retry"
+    done
+}
+
+# TODO: Test withdraw v0 to v1, remove after abandon v0
+# @example withdraw-v0-to-v1 "0x0C1EfCCa2Bcb65A532274f3eF24c044EF4ab6d77" 500
+function withdraw-v0-to-v1() {
+    ethaddr=${1:?"\"$EXECUTABLE withdraw v0 to v1\" requires eth address as 1st argument"}
```
Comment (keroro520) [inline kicker:310]: "withdraw v0 to v1" => "withdraw-v0-to-v1"

**Review by keroro520 — APPROVED** [review]
Look good to me except for some descriptive sentences.

**Review by Flouse — APPROVED** [review]

**Pull request merged**